libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Updaters.hs

```diff
@@ -619,6 +689,27 @@ updatePParams (Alonzo _) pp dpp = case dpp of
   MaxBlockExUnits n -> pp {Alonzo._maxBlockExUnits = n}
   CollateralPercentage perc -> pp {Alonzo._collateralPercentage = perc}
   _ -> pp
+updatePParams (Babbage _) pp dpp = case dpp of
```

Comment:
should we make a comment that we correctly and intentionally are dropping `d` and `extraEntropy` here?

Reply:
this is supposed to be the union over all Eras, so extraEntropy is still around in past Eras.